adr/0002-API-params-naming.md

```diff
@@ -0,0 +1,27 @@
+# 1. Record architecture decisions
+
+Date: 2020-02-06
+
+## Status
+
+Accepted
+
+## Context
+
+The naming of the params used in the API needs be consistent.
+
+For example:
+nomis_offender_id, prison_number and nomis_prison_number refer to the same data.
+
+## Decision
+Use param names similar to those used on [DPS APIs](https://gateway.prod.nomis-api.service.hmpps.dsd.io/elite2api/swagger-ui.html#//offenders/getAddressesByOffenderNoUsingGET)
```

Comment:
The 'DPS API' is usually referred to as 'Elite2' in the MoJ

Reply:
DPS have a number of APIs, of which elite2 is one. To aid comprehension, maybe this could be "Use param names similar to those used on Digital Prison Studio APIs, such as [elite2]"?